service/src/test/java/bio/terra/workspace/common/utils/MockMvcUtils.java

```diff
@@ -164,7 +167,10 @@ public class MockMvcUtils {
 
   public static MockHttpServletRequestBuilder addAuth(
       MockHttpServletRequestBuilder request, AuthenticatedUserRequest userRequest) {
-    return request.header(AUTH_HEADER, "Bearer " + userRequest.getRequiredToken());
+    return request
+        .header(AUTH_HEADER, "Bearer " + userRequest.getRequiredToken())
+        .header(AuthHeaderKeys.OIDC_CLAIM_EMAIL.getKeyName(), userRequest.getEmail())
+        .header(AuthHeaderKeys.OIDC_CLAIM_USER_ID.getKeyName(), userRequest.getSubjectId());
```

Comment:
I think you need to update `UserAccessUtils`, the AuthenticatedUserRequests coming from that (used for connected tests) don't have subject IDs which is breaking tests.

Reply:
thanks!